src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int randomRadius = random.nextInt((100) + 1);
+    private final int randomSide = random.nextInt((100) + 1);
+    private final int randomSideA = random.nextInt((100) + 1);
+    private final int randomSideB = random.nextInt((100) + 1);
+    private final int randomSideC = random.nextInt((100) + 1);
+    private final int randomBaseA = random.nextInt((100) + 1);
+    private final int randomBaseB = random.nextInt((100) + 1);
+    private final int randomFirstLeg = random.nextInt((100) + 1);
+    private final int randomSecondLeg = random.nextInt((100) + 1);
+    private final Figure [] figures = { new Circle(randomRadius),

```

Comment:
you always have the same `Figures`
try to use the `switch-case` in`getRandomFigure()`